src/hotspot/share/logging/logConfiguration.cpp

```diff
@@ -716,3 +718,38 @@ void LogConfiguration::notify_update_listeners() {
 }
 
 bool LogConfiguration::_async_mode = false;
+
+
+#ifdef assert
```

Comment:
That should be ASSERT

Reply:
Oops, I forgot to push that change yesterday.